data/brands/amenity/fuel.json

```diff
@@ -33,6 +33,36 @@
         "name": "1-2-3"
       }
     },
+    {
+      "displayName": "ميد",
+      "locationSet": {"include": ["sa"]},
+      "tags": {
+        "amenity": "fuel",
+        "brand": "ميد",
+        "brand:ar": "شركة ميد التجارية",
+        "brand:en": "MEED Trading Company",
+        "brand:wikidata": "Q105492269",
+        "brand:wikipedia": "ar:شركة ميد",
+        "name": "شركة ميد",
+        "name:ar": "ميد",
+        "name:en": "MEED"
+      }
+    },
+    {
+      "displayName": "نفط",
+      "locationSet": {"include": ["sa"]},
+      "tags": {
+        "amenity": "fuel",
+        "brand": "نفط",
+        "brand:ar": "نفط",
+        "brand:en": "NAFT",
+        "brand:wikidata": "Q105572509",
+        "brand:wikipedia": "ar:شركة نفط (السعودية)",
+        "name": "شركة نفط",
+        "name:ar": "نفط",
```

Comment:
another `name` / `name:ar` mismatch
(I don't really know if it's right or wrong, just that some validators may complain)